packages/web/src/components/user-name-and-badges/UserNameAndBadges.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback } from 'react'
+
+import { User } from '@audius/common'
+
+import { ArtistPopover } from 'components/artist/ArtistPopover'
+import UserBadges from 'components/user-badges/UserBadges'
+import { useGoToRoute } from 'hooks/useGoToRoute'
+import { profilePage } from 'utils/route'
+
+import styles from './UserNameAndBadges.module.css'
+
+export const UserNameAndBadges = ({
+  user,
+  onNavigateAway
+}: {
+  user: User
+  onNavigateAway?: () => void
+}) => {
```

Comment:
I feel like ideally `UserBadges` component should just include the name (like it does on mobile now) but this is fine for now.

Reply:
This probably is least invasive